### ISSUE TITLE
[Suggestion]: Oceanblock "The Skyblock" Island starting items

### ISSUE THREAD

**Issue by xlSoulTaker** [FTBTeam/FTB-Modpack-Issues#102]
### Modpack

OceanBlock

### Is your suggestion related to a problem? Please describe.

tldr; give 5 string in the chest with the saplings and lava bucket

Starting the game with nothing on the island is hard enough when starting on a skyblock island, but the 3% rate in chests is super hard and for people that want to delay the mob farm a bit would still find it a little hard to find string. I tried going around all the wrecks i could see (34) and not a single string dropped. so either up the string probabilities in crates or please add 5 string for the initial mesh and we can make cotton seeds ourselves. [cotton seeds instead of string also appreciated]

### Mod suggestion

n/a
